tcl/db_init.tcl

```diff
@@ -216,4 +216,16 @@ proc qc::db_init {} {
 						    file_id int REFERENCES file(file_id) NOT NULL
 						    );
     }
+
+    # Create anonymous user if none exists
+    qc::db_0or1row {SELECT user_id FROM users WHERE user_id='-1'} {
+	set password_hash [qc::password_hash TheAnonymousUserCreepsSilentlyIntoTheDatabase]
+	qc::db_dml {
+	    INSERT INTO
+	    users (user_id, firstname, surname, email, password_hash)
+	    VALUES
+	    (-1, 'anonymous', 'anonymous', 'anonymous@nowhere.co.uk', :password_hash
+	     );
```

Comment:
End bracket moved to end of previous line.

On 10 August 2016 at 16:34, Bernhard van Woerden <notifications@github.com>
wrote:

> In tcl/db_init.tcl
> <https://github.com/qcode-software/qcode-tcl/pull/412#discussion_r74268685>
> :
>
> > @@ -216,4 +216,16 @@ proc qc::db_init {} {
> >  						    file_id int REFERENCES file(file_id) NOT NULL
> >  						    );
> >      }
> > +
> > +    # Create anonymous user if none exists
> > +    qc::db_0or1row {SELECT user_id FROM users WHERE user_id='-1'} {
> > +	set password_hash [qc::password_hash TheAnonymousUserCreepsSilentlyIntoTheDatabase]
> > +	qc::db_dml {
> > +	    INSERT INTO
> > +	    users (user_id, firstname, surname, email, password_hash)
> > +	    VALUES
> > +	    (-1, 'anonymous', 'anonymous', 'anonymous@nowhere.co.uk', :password_hash
> > +	     );
>
> End brackets would be neater on previous line.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qcode-software/qcode-tcl/pull/412/files/40f3091287fa2ce1658b78d222f090d75462f5a1#r74268685>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS-1fwHebrwRkZXI39iHs_PYP1srYrJUks5qee9ygaJpZM4JhPW9>
> .
>